### PR TITLE
Home task on Module 2

### PR DESCRIPTION
**Pull request by vshokhava**

### CODE REVIEW

File: js-mentoring-program-master/Module2HomeTasks/tasks_condition_condition.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ */
+{
+    const x = 3;
+    const y = 7;
+    let z;
+if (a < 10 && b < 10) {
+    z = a + b;
+} else {
+z = a * b;
+};
```
Comment (dshaplyko) [inline js-mentoring-program-master/Module2HomeTasks/tasks_condition_condition.js]: Be careful with code formatting

File: js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  }
+}
+
+/**
+ * return concatenated string from an array of substring
+ */
+{
+  const substr = ["I", " love", " JS"];
+// MY CODE
+    console.log(substr.join(' '));
```
Comment (dshaplyko) [inline js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js]: Good! But let's use for...of cycle here OR a simple for cycle.

File: js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+ * calculate a total of income of certain person
+ */
+{
+  const personIncomes = {
+    salary: 1985,
+    rent: -600,
+    interestOnDeposit: 250,
+    otherExpences: -300
+  };
+    const sum = personIncomes.salary + personIncomes.rent + personIncomes.interestOnDeposit + personIncomes.otherExpences
```
Comment (dshaplyko) [inline js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js]: Another one, let's use for...in here.

File: js-mentoring-program-master/Module2HomeTasks/tasks_condition_condition.js (hunk 1 above)
Comment (vshokhava) [reply js-mentoring-program-master/Module2HomeTasks/tasks_condition_condition.js]: corrected

File: js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js (hunk 2 above)
Comment (vshokhava) [reply js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js]: corrected

File: js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js (hunk 3 above)
Comment (vshokhava) [reply js-mentoring-program-master/Module2HomeTasks/tasks_cycles_cycles.js]: corrected

**Comment by dshaplyko** [comment]
Accepted, please merge it.

**Pull request merged**